src/build/cargo.rs

```diff
@@ -154,7 +142,8 @@ fn run_cargo(compilation_cx: Arc<Mutex<CompilationContext>>,
         filter: CompileFilter::new(opts.lib,
                                 &opts.bin, opts.bins,
                                 // TODO: Support more crate target types
-                                &[], false, &[], false, &[], false),
+                                &[], false, &[], false, &[], false,
+                                false),
```

Comment:
This is a new `all_targets` option, added in the newer versions of Cargo (using Cargo PR branch meant also using newest Cargo `master`).